frontend/components/shelter/login/SignUpForm.tsx

```diff
@@ -44,6 +44,21 @@ const SignUpForm = ({ onFinish, disableButton }: SignUpFormProps) => (
 				placeholder="Password"
 			/>
 		</Form.Item>
+		<Form.Item
+			name="tempRegistrationKey"
+			rules={[
+				{
+					required: true,
+					message: "Please input a registration key!",
+				},
+			]}
+		>
+			<Input.Password
+				prefix={<LockOutlined className={styles.inputFieldIcons} />}
+				type="tempRegistrationKey"
```

Comment:
`type` prop is for the [input type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Form_%3Cinput%3E_types), can leave it out as the default of `text` should be sufficient

Reply:
updated